src/utils.ts

```diff
@@ -351,7 +351,18 @@ export class Utils {
   static updateScrollResize(event: MouseEvent, el: HTMLElement, distance: number): void {
     const scrollEl = this.getScrollParent(el);
     const height = scrollEl.clientHeight;
-    const offsetTop = scrollEl.getBoundingClientRect().top;
+    //when does pointer move out of visible area, which is then fixed by scrolling that element?

```

Comment:
not sure that first line help - also wrong syntax.
Also please change comments to '// ' (add space)

can we maybe rephrase this comment ? It's not helping me. I would document why we need offsetTop at all, THEN explain that document HTML is different due to returning viewport and not actual needed height. or something like that...

I have not looked into this so taking you word for it. Also add #1745 #1727 to your comment.